packages/office-addin-manifest/test/test.ts

```diff
@@ -552,6 +552,16 @@ describe("Unit Tests", function() {
         assert.strictEqual(validation.report!.warnings!.length, 0);
         assert.strictEqual(validation.report!.addInDetails!.supportedProducts!.length > 0, true);
       });
+      it("valid manifest prod (fail)", async function() {
+        this.timeout(6000);
+        const validation = await validateManifest("test/manifests/TaskPane.manifest.xml", true);
+        assert.strictEqual(validation.isValid, false);
+        assert.strictEqual(validation.status, 200);
+        assert.strictEqual(validation.report!.errors!.length, 2);
```

Comment:
I don't think it is that nice for the tests to be so specific like that. The error length might change in the future.
Could we change this to: `assert.strictEqual(validation.report!.errors!.length > 0, true);`

Reply:
We do want to be specific and check for the number of errors errors (it'ls the length of an array being checked) because we need to know if the validation changes and our template manifest needs to be updated (from what I hear we've been broken before).

Reply:
Makes sense that we are testing if the validation changes